hack/functests.sh

```diff
@@ -37,4 +37,10 @@ if [[ ${KUBEVIRT_PROVIDER} == os-* ]] || [[ ${KUBEVIRT_PROVIDER} =~ (okd|ocp)-*
     oc=${kubectl}
 fi
 
-${TESTS_OUT_DIR}/tests.test -kubeconfig=${kubeconfig} -container-tag=${docker_tag} -container-tag-alt=${docker_tag_alt} -container-prefix=${functest_docker_prefix} -image-prefix-alt=${image_prefix_alt} -oc-path=${oc} -kubectl-path=${kubectl} -gocli-path=${gocli} -test.timeout 420m ${FUNC_TEST_ARGS} -installed-namespace=${namespace} -previous-release-tag=${PREVIOUS_RELEASE_TAG} -previous-release-registry=${previous_release_registry} -deploy-testing-infra=${deploy_testing_infra}
+test_options="-kubeconfig=${kubeconfig} -container-tag=${docker_tag} -container-tag-alt=${docker_tag_alt} -container-prefix=${functest_docker_prefix} -image-prefix-alt=${image_prefix_alt} -oc-path=${oc} -kubectl-path=${kubectl} -gocli-path=${gocli} -test.timeout 420m ${FUNC_TEST_ARGS} -installed-namespace=${namespace} -previous-release-tag=${PREVIOUS_RELEASE_TAG} -previous-release-registry=${previous_release_registry} -deploy-testing-infra=${deploy_testing_infra}"
+if ! [[ ${KUBEVIRT_PROVIDER} =~ .*(k8s-1\.16)|(k8s-1\.17)|k8s-sriov.* ]]; then
+    echo "Will test network in dual stack mode."
+    test_options="$test_options -dual-stack"
```

Comment:
The dual stack flag merits a different PR with different test that just check if cluster is dual stack or not, tests are going to   iterate over the ip addresses so it's transparent for them, let's comment about if script/flag or none of this on that PR that PR.

Reply:
> The dual stack flag merits a different PR with different test that just check if cluster is dual stack or not, tests are going to iterate over the ip addresses so it's transparent for them, let's comment about if script/flag or none of this on that PR that PR.

It is on a separate PR: https://github.com/kubevirt/kubevirt/pull/4139

It's mentioned on the `special notes for your reviewer` section, under `Depends-on:` . 

I agree that the tests should transparently check the VMI's pod IPs and do stuff depending on it. But sometimes, you don't have the VMI yet - as in here, where you're trying to decide the probe spec you'll set up in the VMI spec.

I honestly prefer using a simple flag than checking the state of a known KubeVirt pod (controller ? handler ? ...) and see if it happens to have an IPv6 address.